tasks/install.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: download latest dokuwiki
+  get_url:
+    url: https://download.dokuwiki.org/src/dokuwiki/dokuwiki-stable.tgz
+    dest: /tmp/dokuwiki-stable.tgz
+ 
+- name: unarchive dokuwiki-stable.tgz to {{ dokuwiki_install_path }}
+  unarchive:
+    src: /tmp/dokuwiki-stable.tgz
+    dest: "{{ dokuwiki_install_path | dirname }}"
+    owner: www-data
+    remote_src: yes
+    list_files: yes
+  register: file_list
+
+- name: remove stupid version name from folder path
+  file:
+    src: "{{ dokuwiki_install_path | dirname }}/{{ file_list['files'][0] }}"
+    dest: "{{ dokuwiki_install_path }}"
+    owner: www-data
+    state: link
+
+- name: Install xml-php package used by dokuwiki.
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - php-ldap
+    - php-xml
+  when: ansible_pkg_mgr == "apt"
+
+- name: Copy Logo to Server
+  copy:
+    src: files/logo.png
+    dest: "{{dokuwiki_install_path}}/data/media/wiki/logo.png"
```

Comment:
The task is fine, but the stuvus logo file should not be the default and configurable.

Reply:
done

Reply:
Did you test this? Is `inventory_hostname` the correct var? 
In the readme you talk about a `files` folder.